client/web/pages/stats/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { createFragmentContainer, graphql } from 'react-relay'
+
+import StatsView from '../../components/stats-view'
+import App from '../app'
+
+const StatsContainer = createFragmentContainer(StatsView, graphql`
+  fragment stats on Stats {
+    activeFlights
+    activePassengers
+    topNationalities {
+      code
+      passengers
+    }
+  }
+`)
```

Comment:
This looks a bit weird to me. You should always fetch the data you need in the component. You are fetching things like `activePassengers` and `activeFlights` but you do not use them in the same file. This is confusing. Could you please try to improve this?